api/v1alpha1/hardwareclassification_types.go

```diff
@@ -27,106 +27,81 @@ type HardwareClassificationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// HardwareCharacteristics defines expected hardware configurations for Cpu, Disk, Nic and Ram.
-	HardwareCharacteristics HardwareCharacteristics `json:"hardwareCharacteristics,omitemty"`
```

Comment:
Don't remove this.

Reply:
Revert back to merged changes as per #PR 21.
Files-:
hardwareclassifcation_types.go
metal3.io_hardwareclassifications.yaml

Reply:
Great